andela/isildur-rc
# Feature/151314245/prices in locale currencies

#### What does this PR do?
Displays prices in local currencies

#### Description of Task to be completed?
Converting Prices of products in dollar(default price) to the local currencies of users of the platform and display it to them.

**Business/User Value**
It enables a buyer know the price of products in his or her local currency.
It saves buyers the stress of converting foreign currency to local currency
It help the local payment platform to get the value for commodities in local currencies

**Acceptance Criteria**
GIVEN I am a buyer
WHEN I visit the product page(shop)
THEN Product prices should be displayed in my local currencies.

#### How should this be manually tested?
- enable location on your system
- visit the product page from your local community and you see prices in your local currency

#### Any background context you want to provide?
Nill

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/151314245

#### Screenshots (if appropriate)
<img width="1273" alt="screen shot 2017-10-08 at 4 53 36 pm" src="https://user-images.githubusercontent.com/12856872/31318487-7d7fe83e-ac4b-11e7-802a-a6aa88594a16.png">

<img width="1273" alt="screen shot 2017-10-08 at 4 53 36 pm" src="https://user-images.githubusercontent.com/12856872/31318527-f9088272-ac4b-11e7-9eb1-5f5ca5e9617c.png">
